RELEASE-NOTES.txt

```diff
@@ -983,7 +983,7 @@ o LANG-462:  FastDateFormat supports parse methods.
 BUG FIXES
 ===========
 
-o LANG-932:  Spelling fixes. Thanks to Ville Skytt�.
+o LANG-932:  Spelling fixes. Thanks to Ville Skyttä.
```

Comment:
Interesting issue with these special characters. But I think this file is generated automatically by a script during release, so it may be overridden later I guess? Someone else more familiar with the release process of lang can comment here I think

Reply:
I checked it and found that it should be because the project default encoding is ISO-8859-1. Github seems to use UTF-8 encoding by default to read the content and cause garbled characters.